src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+.head {
+  width: 100%;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  list-style-type: none;
+  height: 15px;
+  padding-right: 50px;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

Comment:
remove paddings, add height to each item

Reply:
I have added some changes, but it doesn`t work without paddings.